src/main/java/ua/goit/gojava32/kickstarter/model/Comment.java

```diff
@@ -0,0 +1,94 @@
+package ua.goit.gojava32.kickstarter.model;
+
+import javax.persistence.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Entity
+@Table (name = "comments")
+public class Comment implements Comparable {
```

Comment:
also I recommend to explicitly use sorting in Dao rather than rely on comparability